docker-compose.yml

```diff
@@ -31,6 +31,9 @@ services:
       HASURA_GRAPHQL_ADMIN_SECRET: ${HASURA_GRAPHQL_ADMIN_SECRET}
       HASURA_EVENT_SECRET: ${HASURA_EVENT_SECRET}
       HASURA_GRAPHQL_AUTH_HOOK: http://${DOCKER_GATEWAY_HOST:-host.docker.internal}:${LOCAL_VERCEL_PORT}/api/hasura/auth
+      HASURA_CRON_BASE_URL: http://host.docker.internal:${LOCAL_VERCEL_PORT}/api/hasura/cron
+      HASURA_EVENT_TRIGGER_BASE_URL: http://host.docker.internal:${LOCAL_VERCEL_PORT}/api/hasura/event-triggers
+      TELEGRAM_BOT_TOKEN: ${TELEGRAM_BOT_TOKEN}
```

Comment:
this shouldn't be required for the event hook to work, its getting from the config directly

Reply:
Right, deleted from this file.